combat-trainer.lic

```diff
@@ -1910,14 +1919,38 @@ class GameState
     @use_weak_attacks = false
   end
 
+  def use_analyze_flame?
+    @use_analyze_flame
+  end
+
+  def stop_analyze_flame
+    @use_analyze_flame = false
+  end
+
   def melee_attack_verb
     if use_weak_attacks?
       brawling? ? 'punch' : 'jab'
+    elsif use_analyze_flame?
+      update_analyze_flame_array if @analyze_flame_array.empty?
+      @analyze_flame_array.shift
     else
       'attack'
     end
   end
 
+  def update_analyze_flame_array
+    result = DRC.bput('analyze flame', 'Analyze what\?', 'by landing a .*')
```

Comment:
This is probably fine, and as GameState advances, it will probably use DRC more.